client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/footer/PlanFooter.tsx

```diff
@@ -1,25 +1,43 @@
 import { translate } from 'i18n-calypso';
+import moment from 'moment';
 import PurchaseDetail from 'calypso/components/purchase-detail';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { useSelector } from 'calypso/state';
-import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSiteOptions, getSiteSlug } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 
+function isMonthsOld( rawDate: string, months: number ) {
```

Comment:
The function looks good; just notice that the second param doesn't have a clear name as `months` is too generic. Maybe something like `ageThresholdInMonths` would be more meaningful.